src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs

```diff
@@ -0,0 +1,425 @@
+            public uint boolValue { get; set; }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class GestureInfoArgs : EventArgs
+        {
+            public GestureInfoType gestureInfo { get; internal set; }
+        }
+
+        private void OnGestureInfoEvent(IntPtr data) {
+            if (data == IntPtr.Zero)
+                return;
+
+            var arg = new GestureInfoArgs();
+            var ges = new GestureInfoType();
+
+            ges.type = (AccessibilityGesture) AccessibilityDoGestureSignal.ConvertParam1(data);
+            ges.xBeg = AccessibilityDoGestureSignal.ConvertParam2(data);
+            ges.xEnd = AccessibilityDoGestureSignal.ConvertParam3(data);
+            ges.yBeg = AccessibilityDoGestureSignal.ConvertParam4(data);
+            ges.yEnd = AccessibilityDoGestureSignal.ConvertParam5(data);
+            ges.state = (GestureState) AccessibilityDoGestureSignal.ConvertParam6(data);
+            ges.eventTime = AccessibilityDoGestureSignal.ConvertParam7(data);
+            ges.boolValue = AccessibilityDoGestureSignal.ConvertParam8(data);
+
+            arg.gestureInfo = ges;
+
+            gestureInfoHandler?.Invoke(this, arg);
+
+            AccessibilityDoGestureSignal.SetResult(data, Convert.ToInt32(ges.boolValue));
+        }
+
+        // This uses DoGestureInfo signal from C++ API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<GestureInfoArgs> GestureInfoReceived {
+            add {
+                if (gestureInfoHandler == null) {
+                    gestureInfoCallback = OnGestureInfoEvent;
+                    GestureInfoSignal().Connect(gestureInfoCallback);
+                }
+                gestureInfoHandler += value;
+            }
+            remove {
+                gestureInfoHandler -= value;
+                if (gestureInfoHandler == null && GestureInfoSignal().Empty() == false)
+                    GestureInfoSignal().Disconnect(gestureInfoCallback);
+            }
+        }
+
+        internal AccessibilityDoGestureSignal GestureInfoSignal() {
+            var handle = GetControl();
+            AccessibilityDoGestureSignal ret = new AccessibilityDoGestureSignal(Interop.ControlDevel.Dali_Toolkit_DevelControl_AccessibilityDoGestureSignal(handle), false);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        ///////////////////////////////////////////////////////////////////
+        // ************** AccessiblityGetDescriptionSignal ************* //
+        ///////////////////////////////////////////////////////////////////
+
+        private delegate void GetDescriptionHandlerType(IntPtr data);
+        private GetDescriptionHandlerType getDescriptionCallback;
+        private EventHandler<GetDescriptionArgs> getDescriptionHandler;
+
+        public class GetDescriptionArgs : EventArgs
```

Comment:
Need to add [EditorBrowsable(EditorBrowsableState.Never)] tag
and solve the build warnings here. : nest type and renaming.

Reply:
Done.